src/components/Board.js

```diff
@@ -0,0 +1,51 @@
+import Square from './Square';
+
+const Board = (props) => {
+  const {squares, onClick, winLine} = props;
+
+  const renderSquare = (i) => {
+    let isWinSquare;
+    if (winLine) {
+      isWinSquare = winLine.indexOf(i) !== -1 ? true : false;
+    } else {
+      isWinSquare = null;
+    }
```

Comment:
```suggestion
    const isWinSquare = winLine && winLine.some(winSquare => winSquare ===  i)
```

letは極力使わないでください。letは再代入可能な変数のため値が変わることを考慮しながら実装しなくてはいけません。constならそんな心配ないので。